packages/vanilla/package.json

```diff
@@ -5,9 +5,27 @@
   "author": "Parallel Markets (https://parallelmarkets.com)",
   "license": "MIT",
   "homepage": "https://github.com/parallel-markets/parallel-js/tree/master/packages/vanilla",
-  "main": "dist/parallelmarkets-vanilla.cjs.js",
-  "module": "dist/parallelmarkets-vanilla.esm.js",
-  "types": "types/index.d.ts",
+  "main": "./dist/parallelmarkets-vanilla.cjs.js",
+  "module": "./dist/parallelmarkets-vanilla.esm.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/parallelmarkets-vanilla.esm.js"
+      },
+      "require": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/parallelmarkets-vanilla.cjs.js"
+      }
+    }
+  },
```

Comment:
The "exports" entry is based on recommendations in this Best Practices post: https://snyk.io/blog/best-practices-create-modern-npm-package/?ck_subscriber_id=1697729601